pages/franchise.md

```diff
@@ -6,15 +6,22 @@ title: Franchise Pages
 
 ```sql franchises
 select 
-Franchise,
-'/franchise_page/' || t.Franchise as franchise_link,
-"Photo URL" as logo
+  Franchise,
+  '/franchise_page/' || t.Franchise as franchise_link,
+  t."Photo URL" as logo,
+  SUM(team_wins)::int || ' - ' || SUM(team_losses)::int as Record,
 from teams t
-order by franchise asc
+  join s17_standings s17
```

Comment:
Can we be more explicit with the join type?

Reply:
changed "join" to "inner join"